dataflow/src/test/java/reachDefs/ReachDefinitions.java

```diff
@@ -0,0 +1,43 @@
+package reachDefs;
```

Comment:
Can we use a consistent package here and in 'org.checkerframework.dataflow.reachdefinitions'? Using upper-case letters in package names is non-standard, I believe? Maybe `reachingdefs`?
(Can you file an issue that we should fix this for the live variable system?)

Reply:
fixed.